bodo/pandas/physical/aggregate.h

```diff
@@ -136,6 +136,54 @@ class PhysicalAggregate : public PhysicalSource, public PhysicalSink {
             /*use_sql_rules*/ false, /* pandas_drop_na_*/ dropna.value());
     }
 
+    explicit PhysicalAggregate(std::shared_ptr<bodo::Schema> in_table_schema,
+                               duckdb::LogicalDistinct& op) {
+        std::map<std::pair<duckdb::idx_t, duckdb::idx_t>, size_t> col_ref_map =
```

Comment:
Refactoring of the duplicated code here would be great.

Reply:
Done.